metricflow-semantics/metricflow_semantics/test_helpers/snapshot_helpers.py

```diff
@@ -301,7 +301,14 @@ def assert_linkable_element_set_snapshot_equal(  # noqa: D103
                         if linkable_metric.join_by_semantic_model
                         else ""
                     ),
-                    tuple(entity_link.element_name for entity_link in linkable_metric.join_path.entity_links),
+                    (
+                        str(tuple(entity_link.element_name for entity_link in linkable_metric.join_path.entity_links)),
```

Comment:
This fixes the symptom but why are they showing up multiple times, anyway? Where's the ambiguity coming from?